app/src/main/java/org/ole/planet/myplanet/ui/team/TeamPagerAdapter.kt

```diff
@@ -33,15 +33,9 @@ class TeamPagerAdapter(fm: FragmentActivity, team: RealmMyTeam?, private val isI
             list.add(MainApplication.context.getString(R.string.tasks))
             list.add(MainApplication.context.getString(R.string.calendar))
             list.add(MainApplication.context.getString(if (isEnterprise) R.string.finances else R.string.courses))
-            if (isEnterprise) {
-                list.add("Reports")
-            }
+            if (isEnterprise) list.add("Reports")
             list.add(MainApplication.context.getString(if (isEnterprise) R.string.documents else R.string.resources))
             list.add(MainApplication.context.getString(if (isEnterprise) R.string.applicants else R.string.join_requests))
-            list.removeAt(0)
-            list.removeAt(0)
-            list.add(1, MainApplication.context.getString(if (isEnterprise) R.string.mission else R.string.plan))
-            list.add(2, MainApplication.context.getString(if (isEnterprise) R.string.team else R.string.members))
```

Comment:
also return these lines. as it affects how a team you are a member of is displayed. I should not see joined member as displayed in the screenshot below and chat should be the first tab
`current` on this branch
![Screenshot_20240712_135637](https://github.com/user-attachments/assets/b98287d5-1127-456d-b952-783449615437)

how it `should` be as in master
![Screenshot_20240712_135453](https://github.com/user-attachments/assets/9fabb9b5-7d3a-49b1-933a-389d7e98fd7d)